### ISSUE TITLE
My OAP component find many Null Exception

### ISSUE THREAD

**Issue by a198720** [apache/incubator-skywalking#2027]
Please answer these questions before submitting your issue.

- Why do you submit this issue?
- [ ] Question or discussion
- [x] Bug
- [ ] Requirement
- [ ] Feature or performance improvement

___
### Question
- What do you want to know?

___
### Bug
- Which version of SkyWalking, OS and JRE?
skyalking: 6.0.0-beta  os: centos7 jre:openJDK 1.8
- Which company or project?

- What happen?
My OAP component find many Null Exception,and I find many .

2018-12-09 08:50:04,524 - org.apache.skywalking.oap.server.receiver.trace.provider.handler.v5.grpc.TraceSegmentServiceHandler - 46 [grpc-default-executor-37] DEBUG [] - receive segment
2018-12-09 08:50:04,524 - org.apache.skywalking.oap.server.core.cache.ServiceInventoryCache - 90 [grpc-default-executor-37] DEBUG [] - Get service by id 1 from cache
2018-12-09 08:50:04,524 - org.apache.skywalking.oap.server.core.cache.ServiceInventoryCache - 90 [grpc-default-executor-37] DEBUG [] - Get service by id 6 from cache
2018-12-09 08:50:04,525 - org.apache.skywalking.oap.server.core.remote.selector.ForeverFirstSelector - 35 [DataCarrier.default.Consumser.0.Thread] DEBUG [] - clients size: 1
2018-12-09 08:50:04,525 - org.apache.skywalking.oap.server.core.remote.selector.ForeverFirstSelector - 35 [DataCarrier.default.Consumser.0.Thread] DEBUG [] - clients size: 1
2018-12-09 08:50:04,526 - org.apache.skywalking.oap.server.core.remote.selector.ForeverFirstSelector - 35 [DataCarrier.default.Consumser.0.Thread] DEBUG [] - clients size: 1
2018-12-09 08:50:04,526 - org.apache.skywalking.oap.server.core.remote.selector.ForeverFirstSelector - 35 [DataCarrier.default.Consumser.0.Thread] DEBUG [] - clients size: 1
2018-12-09 08:50:04,526 - org.apache.skywalking.oap.server.core.analysis.worker.IndicatorAggregateWorker - 81 [DataCarrier.IndicatorAggregateWorker.instance_jvm_old_gc_count.Consumser.0.Thread] DEBUG [] - org.apache.skywalking.oap.server.core.analysis.generated.serviceinstancejvmgc.InstanceJvmOldGcCountIndicator@fcf065c0
2018-12-09 08:50:04,527 - org.apache.skywalking.oap.server.core.cache.ServiceInventoryCache - 108 [grpc-default-executor-37] DEBUG [] - service id 6 not find in cache.
2018-12-09 08:50:04,527 - org.apache.skywalking.oap.server.receiver.trace.provider.parser.SegmentParseV2 - 100 [grpc-default-executor-37] ERROR [] - null
java.lang.NullPointerException: null
2018-12-09 08:50:04,530 - org.apache.skywalking.oap.server.core.analysis.worker.IndicatorAggregateWorker - 81 [DataCarrier.IndicatorAggregateWorker.instance_jvm_memory_noheap_max.Consumser.0.Thread] DEBUG [] - org.apache.skywalking.oap.server.core.analysis.generated.serviceinstancejvmmemory.InstanceJvmMemoryNoheapMaxIndicator@fcf065c0
2018-12-09 08:50:04,531 - org.apache.skywalking.oap.server.core.analysis.worker.IndicatorAggregateWorker - 81 [DataCarrier.IndicatorAggregateWorker.instance_jvm_young_gc_time.Consumser.0.Thread] DEBUG [] - org.apache.skywalking.oap.server.core.analysis.generated.serviceinstancejvmgc.InstanceJvmYoungGcTimeIndicator@fcf065c0
2018-12-09 08:50:04,544 - org.apache.skywalking.oap.server.core.storage.PersistenceTimer - 81 [pool-10-thread-1] DEBUG [] - extract org.apache.skywalking.oap.server.core.analysis.worker.IndicatorPersistentWorker worker data size: 8
2018-12-09 08:50:04,545 - org.apache.skywalking.oap.server.core.storage.PersistenceTimer - 74 [pool-10-thread-1] DEBUG [] - extract org.apache.skywalking.oap.server.core.analysis.worker.IndicatorPersistentWorker worker data and save
2018-12-09 08:50:04,565 - org.apache.skywalking.oap.server.core.storage.PersistenceTimer - 81 [pool-10-thread-1] DEBUG [] - extract org.apache.skywalking.oap.server.core.analysis.worker.IndicatorPersistentWorker worker data size: 8
2018-12-09 08:50:04,566 - org.apache.skywalking.oap.server.core.storage.PersistenceTimer - 74 [pool-10-thread-1] DEBUG [] - extract org.apache.skywalking.oap.server.core.analysis.worker.IndicatorPersistentWorker worker data and save
2018-12-09 08:50:04,587 - org.apache.skywalking.oap.server.receiver.jvm.provider.handler.JVMMetricReportServiceHandler - 47 [grpc-default-executor-37] DEBUG [] - receive the jvm metric from service instance, id: 6
___
### Requirement or improvement
- Please describe about your requirements or improvement suggestions.

**Comment by wu-sheng** [comment]
I still don't understand why two. See our changelog, https://github.com/apache/incubator-skywalking/pull/2028 . These the bugs and features in next release, which is comming soon.

**Comment by wu-sheng** [comment]
I don't understand, why 8 hours. In 6.x series, we didn't do anything in 8 hours cycle. In 5.x, deleting data did, which in some cases could make your ElasticSearch blocking. I used to know from the community feedback.

**Comment by a198720** [comment]
I am also thinking about why it is 8 hours.  I looked for all the configurations and found no 8 hour related configuration. The only thought is that the Chinese time zone and UTC are 8 hours apart. But when I don't use namespace, I can run normally. But in the case of not using namespace, it is working fine. So I  have been understanding the source code of the storage-elasticsearch-plugin component and I hope I can find the reason.

**Comment by wu-sheng** [comment]
Are you using 5.x? In 5.x, first delete happens in 8 hours later.

**Comment by a198720** [comment]
No, I user 6.0.0-beta. I checkout  the source code from master, and build it myself.

**Comment by wu-sheng** [comment]
Then, no clue.
